### PR TITLE
docs(ARCHITECTURE): Warning for DMMF

### PR DESCRIPTION
**Pull request by janpio**
Added small warning that API is internal only, no guarantees.

### CODE REVIEW

File: ARCHITECTURE.md [hunk 1]
```diff
@@ -24,16 +24,18 @@ See [Prisma Generators](https://prismaio.notion.site/Prisma-Generators-a2cdf2622
 
 ## The `DMMF`, or Data Model Meta Format
 
-What the ... is DMMF? It's the Datamodel Meta Format. It is an AST (abstract syntax tree) of the datamodel in the form of JSON.
+What the ... is DMMF? It's the Datamodel Meta Format. It is an AST (abstract syntax tree) of the datamodel in the form of JSON.  
 The whole Prisma Client is just generated based on the DMMF, which comes from the Rust engines.
 
+> ⚠️ Note:  DMMF is a Prisma ORM internal API with no guarantees for stability to outside users. We might - and do - change the DMMF in potentially breaking ways between minor versions. 🐲
```
Comment (millsp) [inline ARCHITECTURE.md:30]:
```suggestion
> ⚠️ Note: The DMMF is a Prisma ORM internal API with no guarantees for stability to outside users. We might - and do - change the DMMF in potentially breaking ways between minor versions. 🐲
```

File: ARCHITECTURE.md [hunk 2]
```diff
@@ -24,16 +24,18 @@ See [Prisma Generators](https://prismaio.notion.site/Prisma-Generators-a2cdf2622
 
 ## The `DMMF`, or Data Model Meta Format
 
-What the ... is DMMF? It's the Datamodel Meta Format. It is an AST (abstract syntax tree) of the datamodel in the form of JSON.
+What the ... is DMMF? It's the Datamodel Meta Format. It is an AST (abstract syntax tree) of the datamodel in the form of JSON.
```
Comment (millsp) [inline ARCHITECTURE.md:27]: Is the double space intended?

**Review by millsp — CHANGES_REQUESTED** [review]

File: ARCHITECTURE.md (hunk 2 above)
Comment (Jolg42) [reply ARCHITECTURE.md:27]: Probably yes to have a line return

**Review by Jolg42 — APPROVED** [review]
Thanks!

File: ARCHITECTURE.md (hunk 2 above)
Comment (janpio) [reply ARCHITECTURE.md:27]: Yep, the document was written as if someone originally intended there to be line breaks, but GitHub Markdown does only render them with double spaces - so I added them. Alternative would have been to reformat the file itself to not have line breaks that do not render.

**Comment by janpio** [comment]
Hmm, why does the `Detect jobs to run` job not run here but is expected to pass?
I could merge this with admin rights not, but do we have a problem here in our config, maybe because this just edits a `.md` file maybe?

**Comment by Jolg42** [comment]
@janpio I think the current issue with the setup is that "Detect jobs to run" is a required step (useful so automerge doesn't merge before it runs)

But here it cannot run because it's skipped by `paths-ignore` https://github.com/prisma/prisma/blob/main/.github/workflows/test.yml#L32

So maybe we can improve later by doing the logic of skipping in "Detect jobs to run" so it is never skipped

**Comment by Jolg42** [comment]
By the way should we merge this now?

**Review by millsp — APPROVED** [review]

**Review by aqrln — APPROVED** [review]
To me it doesn't look like those paragraphs were necessarily intended to be rendered with line breaks, some people just prefer putting each sentence on its own line in Markdown sources instead of hard-breaking lines at a specific column, but LGTM either way.

**Comment by janpio** [comment]
Abused admin rights to merge. Please create an issue to fix the underlying issue as it could otherwise block us in the future.

**Comment by aqrln** [comment]
@janpio https://github.com/prisma/prisma/issues/10700

**Pull request merged**